Parser/asdl_c.py

```diff
@@ -1244,6 +1244,30 @@ def visitModule(self, mod):
             Py_DECREF(unused);
         }
     }
+
+    // Discard fields from 'expecting' that default to None
```

Comment:
Is it possible to have default fields that are not None?

Reply:
Not in the core.